src/main/java/org/jenkinsci/plugins/ewm/definitions/DiskPool.java

```diff
@@ -25,14 +25,14 @@
 public class DiskPool implements Describable<DiskPool> {
 
     private final String diskPoolId;
-    private final String name;
+    private final String displayName;
     private final String description;
     private final List<Disk> disks;
 
     @DataBoundConstructor
-    public DiskPool(String diskPoolId, String name, String description, List<Disk> disks) {
+    public DiskPool(String diskPoolId, String displayName, String description, List<Disk> disks) {
```

Comment:
Not for this PR. It would be great to prevent null IDs from the API complexity PoV